content/how-to/account-management/account-registration-code/index.md

```diff
@@ -1,26 +1,22 @@
 ---
 permalink: account-registration-code
```

Comment:
This permalink and the account registration hyperlink take me to the page below.
<img width="1680" alt="Screen Shot 2022-09-26 at 9 39 34 PM" src="https://user-images.githubusercontent.com/14296817/192418857-d21a1813-6154-4941-bb14-d68450e341e6.png">

So it seems to be going to the other site. Look for a redirect and fix if needed.

Reply:
Need to discuss.

Reply:
Done